src/MoBi.Presentation/Presenter/BuildingBlockWithInitialConditionsPresenter.cs

```diff
@@ -0,0 +1,129 @@
+using System;
```

Comment:
This is a new base class that has no new code, only pulled up from the former InitialConditionsTask